upgrade-1.x-to-2.x.sh

```diff
@@ -990,7 +990,16 @@ if [ "$NOREBOOT" == "no" ]; then
     # Reboot into new OS
     log "Rebooting into new OS in 5 seconds..."
     progress 100 "Update successful, rebooting"
-    nohup bash -c " /bin/sleep 5 ; /sbin/reboot " > /dev/null 2>&1 &
+    systemd-run --on-active=5 --unit=hup-reboot.service systemctl reboot
+    sleep 300
+    # If the previous reboot command has failed for any reason, let's try differently
+    nohup bash -c "reboot --force" > /dev/null 2>&1 &
```

Comment:
why not schedule this as a unit as well? does that somehow block the reboot process?

Reply:
@xginn8 if you check the thread with @ZubairLK , there the idea was: "if a systemctl reboot fails, [we might assume] we can't rely on systemd anymore."

Reply:
But yeah, doing the same as the first step just with `--force` or `--force --force` was also on the list. This is more "defence in depth", I guess.